next.config.js

```diff
@@ -7,6 +11,70 @@ const nextConfig = {
   }
 }
 
+// API endpoints that should be forwarded to the legacy site
+const legacyAPIPaths = [
+  '/total-ogn',
+  '/total-ogv',
+  '/total-ousd',
+  '/circulating-ogn',
+  '/circulating-ogv',
+  '/social-stats',
+  '/mailing-list/:action*',
+]
+
+// Map legacy routes to new ones here
+const legacyPageMappings = [
+  // [source, destination]
+  ['/dashboard', '/'],
+  ['/presale', '/'],
+  ['/tokens', '/'],
+  
+  ['/product', '/'],
+  ['/ogn-token', '/'],
+  
+  ['/investors', '/community'],
+  ['/team', '/community'],
+  ['/about', '/community'],
+
+  ['/discord', `${NEXT_PUBLIC_DISCORD_URL}`],
+  ['/telegram', `${NEXT_PUBLIC_TELEGRAM_URL}`],
+
+  // Other dead paths:
+  ['/apple-app-site-association', '/'],
+  ['/mobile/apk', '/'],
+  ['/whitepaper.pdf', '/'],
+  ['/ios', '/'],
+  ['/android', '/'],
+  ['/lupefiasco', '/'],
+  ['/dapp', '/'],
+  ['/rewards', '/'],
+  ['/whitepaper', '/'],
+  ['/developers', '/'],
+  ['/privacy/extension', '/'],
+  ['/nft-terms', '/'],
```

Comment:
It would probably be a good idea to maintain this one, if possible.

https://www.originprotocol.com/nft-terms

There may be legacy NFT storefronts pointing to this.